srcpkgs/python3-plotly/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'python3-plotly'
 pkgname=python3-plotly
-version=4.9.0
-revision=2
+version=4.13.0
+revision=1
 wrksrc="plotly-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-setuptools python3-six python3-requests python3-pytz
```

Comment:
Why remove setuptools?

Reply:
As far as I can tell it's only required on the host when building the package. So it should only be in `hostmakedepends`, right?